.github/workflows/master.yml

```diff
@@ -2,21 +2,40 @@
 name: CI
 on:
   push:
-    branches: master
+    branches: ['master']
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ secrets.IMAGE_NAME }}
```

Comment:
What do I need to add to the secrets here?

Reply:
Just add a repository secret with the image name you want. From what I understand of your current setup, this would be `tum-dev/calendarproxy/server`

Reply:
Ah, yes, easy enough. Thanks, that did the trick.